_data/devdocs/en/references/p2p_networking.md

```diff
@@ -1432,6 +1432,11 @@ The following service identifiers have been assigned.
 |-------|--------------|---------------
 | 0x00  | *Unnamed*    | This node is not a full node.  It may not be able to provide any data except for the transactions it originates.
 | 0x01  | NODE_NETWORK | This is a full node and can be asked for full blocks.  It should implement all protocol features available in its self-reported protocol version.
+| 0x02  | NODE_GETUTXO | This is a full node capable of responding to the getutxo protocol request. See BIP-64 for details on how this is implemented.
+| 0x04  | NODE_BLOOM | This is a full node capable and willing to handle bloom-filtered connections.
+| 0x08  | NODE_WITNESS | This is a full node that can be asked for blocks and transactions including witness data.
+| 0x10  | NODE_XTHIN | This is a full node that supports Xtreme Thinblocks.
```

Comment:
Should probably mention that this is not supported by any currently-maintained Bitcoin full node.  The Bitcoin Unlimited website [says](https://www.bitcoinunlimited.info/download) the last version for the BTC network was in 2017.

Reply:
Is there any decent link to be added for this one?

Reply:
I guess this, but it seems quite underspecified to me: https://github.com/BitcoinUnlimited/BUIP/blob/master/010.mediawiki